openmetadata-ui/src/main/resources/ui/src/generated/entity/data/glossaryTerm.ts

```diff
@@ -186,11 +190,11 @@ export interface EntityReference {
 
 export interface TermReference {
   /**
-   * Name that identifies the source of an external glossary term. Example `HealthCare.gov`
+   * Name that identifies the source of an external glossary term. Example `HealthCare.gov`.
```

Comment:
This should be endpoint

Reply:
These are auto generated file from script, we do not change it manually